packages/vue-client/src/views/GameView.vue

```diff
@@ -256,16 +270,29 @@ pre {
   text-align: left;
 }
 
-.nav-buttons {
+.round-nav-buttons {
   height: fit-content;
   display: flex;
   flex-direction: row;
+  width: inherit;
+  max-width: var(--board-side-length);
   justify-content: space-between;
+  .left-container {
+    display: flex;
+    flex-direction: row;
+    gap: 10px;
+  }
+  .right-container {
+    display: flex;
+    flex-direction: row;
+    gap: 10px;
+  }
 }
 
 @media (min-width: 664px) {
   .board {
-    width: 600px;
+    width: var(--board-side-length);
+    height: var(--board-side-length);
```

Comment:
I must not have been paying attention before, but TIL about these custom properties!  Cool!